src/bootstrap/run.rs

```diff
@@ -0,0 +1,45 @@
+use crate::builder::{Builder, RunConfig, ShouldRun, Step};
+use crate::tool::Tool;
+use std::process::Command;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct ExpandYamlAnchors;
+
+impl Step for ExpandYamlAnchors {
+    type Output = ();
+    const DEFAULT: bool = true;
+    const ONLY_HOSTS: bool = true;
```

Comment:
This is probably spurious, right? I presume that the run step here is only called when explicitly invoked, not automatically (since `x.py run` requires an argument).

So we'll not actually want default or only_hosts here. It might not even make sense to use `Step`s for this at all but we can leave that be for the time being.

Reply:
Yes, fixed it. I'm not *too* familiar with bootstrap so I did it the same way as the other steps I was seeing around.